Lib/profiling/sampling/_vendor/d3-flame-graph/4.1.3/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Do we need to consult with PSF legal about distributing these? Based on https://docs.python.org/3/license.html, it looks we already distribute an Apache-2 licensed dependency so I expect the answer is "probably not"?

Also I think https://github.com/python/cpython/blob/main/Doc/license.rst is maybe a better place to include this, since most other dependencies/vendored code is listed there.

Reply:
> Do we need to consult with PSF legal about distributing these? Based on https://docs.python.org/3/license.html, it looks we already distribute an Apache-2 licensed dependency so I expect the answer is "probably not"?

Indeed, we have vendored many other MIT and Apache without any problems but just to be sure I will consult with @ambv as the PSF legal if needed.

> Also I think https://github.com/python/cpython/blob/main/Doc/license.rst is maybe a better place to include this license, since most other dependencies/vendored code is listed there.

Excellent point👍